- Remove some flags that gcc 4.2 (on tier 2 archs like powerpc)
  does not recognize
- Do not install the Linux-only font

--- Makefile.am.orig	2018-09-10 08:31:56 UTC
+++ Makefile.am
@@ -14,10 +14,8 @@ cava_LDADD = -liniparser
 cava_LDFLAGS = -L/usr/local/lib -Wl,-rpath /usr/local/lib
 cava_CPPFLAGS = -DPACKAGE=\"$(PACKAGE)\" -DVERSION=\"$(VERSION)\" \
            -D_POSIX_SOURCE -D _POSIX_C_SOURCE=200809L
-cava_CFLAGS = -std=c99 -Wall -Wextra -Wno-unused-result -Wno-maybe-uninitialized 
+cava_CFLAGS = -std=c99 -Wall -Wextra
 
-cava_font_dir = @FONT_DIR@
-cava_font__DATA = cava.psf
 
 if !SYSTEM_LIBINIPARSER
     cava_SOURCES += iniparser/libiniparser.la
